core/common/src/RpcInterface.ts

```diff
@@ -54,14 +59,117 @@ export abstract class RpcInterface {
     const parametersCompat = (arguments.length === 1 && typeof (parameters) === "object") ? parameters : arguments;
     const parametersArray = Array.isArray(parametersCompat) ? parametersCompat : Array.prototype.slice.call(parametersCompat);
     const operationName = parametersArray.pop();
-    const request = new (this.configuration.protocol.requestType as any)(this, operationName, parametersArray) as RpcRequest;
-    request.submit(); // eslint-disable-line @typescript-eslint/no-floating-promises
-    (this as any)[CURRENT_REQUEST] = request;
-    return request.response;
+
+    const session = IpcSession.active;
+    if (session && !(this.configuration instanceof RpcDefaultConfiguration)) {
```

Comment:
Why do you also need to check `!(this.configuration instanceof RpcDefaultConfiguration)`?

Reply:
It was for a test failure at some point, but it looks like I fixed it more deeply with a different change along the way.
Took it out...